test/framework/easyconfig.py

```diff
@@ -4408,6 +4413,10 @@ def test_det_copy_ec_specs(self):
             self.assertEqual(paths, args[:-1])
             self.assertEqual(target_path, args[-1])
 
+        if self.skip_github_tests:
+            print("Skipping test_det_copy_ec_specs using --from-pr, no GitHub token available?")
+            return
```

Comment:
```suggestion
            self.skipTest("Not using --from-pr, no GitHub token available?")
```

See my longstanding work at #3790
```

Reply:
Agreed